man/man7/hashbang.7

```diff
@@ -144,112 +102,22 @@ includes many features of the C shell.
-
 units - Unit conversion utility.
 
-dc - A reverse-polish desk calculator which supports arbitrary-precision
-arithmetic.
-
-qalc - A small simple to use command-line calculator.
-
 bc - An arbitrary precision calculator language
 
 .SS Chat / IM
 weechat-curses - Wee Enhanced Environment for Chat (Curses version)
 
 irssi - A text-based IRC client written in the C programming language.
-
-centerim - A text based menu and window-driven instant messaging interface that
-supports ICQ, Yahoo!, AIM, MSN, IRC, XMPP, LiveJournal, and Gadu-Gadu
-protocols.
-
-finch - A console-based instant messaging client based on the libpurple
-library.
-
-bitlbee - Bitlbee brings Instant Messaging to IRC clients. It has support for 
-multiple IM networks/protocols including Google Talk.
-
-.RS
-To use bitlbee in weechat enter
-.RS
-.B
-/server add bitlbee irc.hashbang.sh/6610
-.RE
-then
-.RS
-.B
-/connect bitlbee 
-.RE
-this will force join you into the 
-.B
-&bitlbee
-channel. If you are interested in using Google Talk follow this guide 
-http://wiki.bitlbee.org/HowtoGtalk
-.RE
-
-barnowl - A curses-based instant messaging client forked from the ktools owl
-project that supports Zephyr, AIM, Jabber, IRC, and Twitter.
-
-pork - An ncurses-based AOL instant messenger and IRC client.
-
-scrollz - An advanced ircII-based IRC client.
-
-ii - A minimalistic FIFO and filesystem based IRC client.
-
-sic - A fast, simple IRC client that reads from standard input and prints
-server output to standard output.
-
-erc - An IRC client integrated into GNU Emacs.
-.SS News Reader
-newsbeuter - A text-based news aggregator that supports RSS and Atom.
-
-rsstail - An RSS news reader which works similar to the tail command.
-
-canto - An ncurses RSS reader built to be flexible on top of python.
-
-rawdog - (RSS Aggregator Without Delusions Of Grandeur) An RSS aggregator that
-uses the Python feedparser module to retrieve articles from feeds in RSS, Atom
-and other formats.
 .SS Web Browsing
-links - A menu-driven text mode web browser with ncurses interface supporting colors, table
-rendering and background downloading.
-
 elinks - Similar to links, but also supports Form Input, Password Management,
 and Tabbed Browsing
 
 lynx - A general purpose distributed information web browser.
 
 w3m - A text based web browser and pager.
-
-html2text - Reads an HTML document and outputs plain text characters.
-.SS Database
-sqlite3 - A relational database management system contained in a C programming
-library.
-
-mariadb - A fork of the MySQL relational database management system led by the
-original developer of MySQL.
-
-postgresql - An Object-relational database management system that emphasises
-extensibility and standards-compliance.
-
-mongodb - A document-oriented NoSQL database that uses JSON-like documents with
-dynamic schemas.
-
-redis - A networked, in-memory, key-value data store with optional durability
```

Comment:
you win. I think I tried to tab-complete `redis` and it found nothing, but now it does indeed find them :)